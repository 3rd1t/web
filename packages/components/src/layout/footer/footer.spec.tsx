import React from "react"
import Footer from "./footer"
import { render } from "@testing-library/react"

describe(" Footer", () => {
  it("renders correctly", () => {
    const { baseElement } = render(<Footer bg="bg-carbon-900" primaryText="text-white" secondaryText="text-gray-200" />)

    expect(baseElement).toMatchSnapshot()
  })
})
