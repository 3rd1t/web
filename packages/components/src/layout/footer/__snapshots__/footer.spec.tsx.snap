// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Footer renders correctly 1`] = `
<body>
  <div>
    <footer
      class="bg-carbon-900"
    >
      <div
        class="container mx-auto"
      >
        <div
          class="flex flex-wrap justify-between p-10  text-gray-200"
        >
          <div
            class="p-2 md:space-y-2"
          >
            <span
              class="text-xs font-semibold text-white uppercase"
            >
              Projects
            </span>
            <a
              class="block focus:outline-none text-gray-200 hover:text-white"
              href="https://perfol.io"
            >
              Home
            </a>
            <div
              class="flex items-center space-x-2"
            >
              <a
                class="block focus:outline-none text-gray-200 hover:text-white"
                href="https://analytics.perfol.io"
              >
                Analytics
              </a>
              <span
                class="px-1 text-sm bg-purple-800 rounded-sm"
              >
                WIP
              </span>
            </div>
            <a
              class="block focus:outline-none text-gray-200 hover:text-white"
              href="https://research.perfol.io"
            >
              Research
            </a>
          </div>
          <div
            class="p-2 md:space-y-2"
          >
            <span
              class="text-xs font-semibold text-white uppercase"
            >
              Resources
            </span>
            <a
              class="block focus:outline-none text-gray-200 hover:text-white"
              href="/docs"
            >
              Documentation
            </a>
            <a
              class="block focus:outline-none text-gray-200 hover:text-white"
              href="perfol.io/roadmap"
            >
              Roadmap
            </a>
            <a
              class="block focus:outline-none text-gray-200 hover:text-white"
              href="/imprint"
            >
              Imprint
            </a>
            <a
              class="block focus:outline-none text-gray-200 hover:text-white"
              href="/privacy"
            >
              Privacy
            </a>
          </div>
          <div
            class="p-2 md:space-y-2"
          >
            <span
              class="text-xs font-semibold text-white uppercase"
            >
              Contact us
            </span>
            <a
              class="block focus:outline-none text-gray-200 hover:text-white"
              href="mailto:info@perfol.io"
            >
              contact@perfol.io
            </a>
          </div>
          <div
            class="p-2 md:space-y-2"
          >
            <span
              class="text-xs font-semibold text-white uppercase"
            >
              Source
            </span>
            <a
              class="block focus:outline-none text-gray-200 hover:text-white"
              href="https://github.com/perfolio"
            >
              github.com
            </a>
          </div>
        </div>
      </div>
      <div
        class="pt-2 border-t border-carbon-700"
      >
        <div
          class="max-w-6xl mx-auto"
        >
          <div
            class="flex flex-col items-center px-10 py-2 text-sm md:flex-row"
          >
            <span
              class="text-white"
            >
              Â© Copyright 
              2020
              . All Rights Reserved.
            </span>
            <div
              class="flex flex-row md:flex-auto md:flex-row-reverse"
            >
              <a
                href="https://www.netlify.com"
              >
                <img
                  alt="Deploys by Netlify"
                  src="https://www.netlify.com/img/global/badges/netlify-light.svg"
                />
              </a>
            </div>
          </div>
        </div>
      </div>
    </footer>
  </div>
</body>
`;
